web/auth/session.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
+	"errors"
+	"net/http"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/cozy/cozy-stack/couchdb"
+	"github.com/cozy/cozy-stack/instance"
+	"github.com/cozy/cozy-stack/web/middlewares"
+	"github.com/gin-gonic/gin"
+)
+
+// SessionsType : The couchdb type for a session
+const SessionsType = "io.cozy.sessions"
+
```

Comment:
@jinroh no, most `io.cozy` types will be used via `/data`, like contacts, calendars, settings, etc.